app.js

```diff
@@ -0,0 +1,110 @@
+// import fetch library to fetch the data from API
+const fetch = require('node-fetch');
```

Comment:
This PR should contain `CustomDate` and `stack` only, I don't want to see any other files